### PR TITLE
Add logs for coinbase

### PR DESCRIPTION
**Pull request by KaustubhShamshery**
## Description

Add logs to tell if the current node has received some reward

### Implementation

- [x] **ready for review**

### Integration Test (Core Team)

- [x] local machine test

- [x] small-scale cloud test

### CODE REVIEW

File: src/libDirectoryService/Coinbase.cpp [hunk 1]
```diff
@@ -301,6 +301,11 @@ void DirectoryService::InitCoinbase() {
                   addr, genesisAccount, reward_each)) {
             LOG_GENERAL(WARNING, "Could Not reward " << addr);
           } else {
+            if (addr ==
+                Account::GetAddressFromPublicKey(m_mediator.m_selfKey.second)) {
+              LOG_GENERAL(INFO, "[REWARD]"
+                                    << "Woohoo! Rewarded " << reward_each);
```
Comment (kaikawaliu) [inline src/libDirectoryService/Coinbase.cpp]: Woohoo!

File: src/libMessage/Messenger.cpp [hunk 2]
```diff
@@ -1938,6 +1938,41 @@ bool Messenger::SetAccountStoreDelta(vector<unsigned char>& dst,
   return SerializeToArray(result, dst, offset);
 }
 
+bool Messenger::StateDeltaToAddressMap(
+    const vector<unsigned char>& src, const unsigned int offset,
+    unordered_map<Address, int>& accountMap) {
```
Comment (kaikawaliu) [inline src/libMessage/Messenger.cpp]: int is too small for this case

Comment (jiayaoqijia) [reply src/libMessage/Messenger.cpp]: @KaustubhShamshery Can you fix this issue now?

File: src/libNode/FinalBlockProcessing.cpp [hunk 3]
```diff
@@ -714,6 +714,23 @@ bool Node::ProcessFinalBlock(const vector<unsigned char>& message,
...
+      LOG_GENERAL(WARNING, "Messenger::StateDeltaToAccountMap failed");
+    } else {
+      auto it = addressMap.find(
+          Account::GetAddressFromPublicKey(m_mediator.m_selfKey.second));
+      if (it != addressMap.end()) {
+        LOG_GENERAL(INFO, "[REWARD]"
+                              << " Got " << it->second << " as reward");
+      } else {
+        LOG_GENERAL(INFO, "[REWARD]"
+                              << "Got no reward thist ds epoch");
```
Comment (Gnnng) [inline src/libNode/FinalBlockProcessing.cpp]: maybe a typo here

```suggestion
                              << "Got no reward this ds epoch");
```

**Pull request merged**